airbyte-integrations/connectors/destination-elasticsearch/src/main/java/io/airbyte/integrations/destination/elasticsearch/ElasticsearchConnection.java

```diff
@@ -58,7 +58,11 @@ public ElasticsearchConnection(ConnectorConfiguration config) {
 
     // Create the low-level client
     httpHost = HttpHost.create(config.getEndpoint());
-    final RestClientBuilder builder = RestClient.builder(httpHost);
+    if (config.getPathPrefix != null) {
```

Comment:
It seems config is not accessible here. @fkkhan2009 please take a look at tests.